e2e/cypress/integration/messaging/emoji_to_markdown_spec.js

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+function createMessages(message) {
+    const aliasLineMessage = 'newLineMessage';
+    const aliasLineMessageReplyInRHS = 'newLineMessageReplyInRHS';
+
+    cy.postMessage(message);
+    cy.getLastPostId().then((postId) => {
+        cy.get(`#postMessageText_${postId}`).as(aliasLineMessage);
```

Comment:
You can omit setting the alias here as it's not used.

Reply:
done.